tools/tool-env/isopackets.js

```diff
@@ -229,8 +229,6 @@ var newIsopacketBuildingCatalog = function () {
       isopacketCatalog.initialize({
         localPackageSearchDirs: [
           files.pathJoin(files.getCurrentToolsDir(), 'packages'),
-          files.pathJoin(files.getCurrentToolsDir(),
-            'packages', 'non-core', 'blaze', 'packages')
```

Comment:
Unfortunately this commit didn't work. I'll push a follow-up commit that fixes this (without hard-coding `blaze`).